pygfx/objects/_more.py

```diff
@@ -4,21 +4,73 @@
 
 
 class Group(WorldObject):
-    """A Group is almost identical to a WorldObject. Its purpose is
-    to make working with groups of objects syntactically clearer.
+    """A group of objects.
+
+    A Group is useful when manipulating the scene graph as children can be
+    jointly moved/scaled/rotated. It typically has no visual properties.
+
+    Parameters
+    ----------
+    geometry : Geometry
+        The data defining the shape of the object.
+    material : Material
+        The data defining the appearence of the object.
+    visible : bool
+        Whether the object is visible.
+    render_order : int
+        The render order (when applicable for the renderer's blend mode).
+    render_mask : str
+        Determines the render passes that the object is rendered in. It's
+        recommended to let the renderer decide, using "auto".
+    position : Vector
+        The position of the object in the world. Default (0, 0, 0).
+
     """
 
 
 class Scene(Group):
-    """The scene is a WorldObject that represents the root of a scene graph.
-    It hold certain attributes that relate to the scene, such as the background color,
-    fog, and environment map. Camera's and lights can also be part of a scene.
+    """Root of the scene graph.
+
+    Parameters
+    ----------
+    geometry : Geometry
+        The data defining the shape of the object.
+    material : Material
+        The data defining the appearence of the object.
+    visible : bool
+        Whether the object is visible.
+    render_order : int
+        The render order (when applicable for the renderer's blend mode).
+    render_mask : str
+        Determines the render passes that the object is rendered in. It's
+        recommended to let the renderer decide, using "auto".
+    position : Vector
+        The position of the object in the world. Default (0, 0, 0).
+
+    The scene holds scene-level information (background color, fog, environment
+    map) as well as all objects that take part in the rendering process as either
+    direct or indirect children/nested objects.
+
     """
 
 
 class Background(WorldObject):
-    """An object representing a scene background.
+    """The scene's background.
+
+    @almarklein: how come this is an independent object and not part of Scene?
+
     Can be e.g. a gradient, a static image or a skybox.
+
+    Parameters
+    ----------
+    geometry : Geometry
+        The geometry to use when rendering the background.
```

Comment:
Seeing this in the code now, it looks like we allow `geometry` for backwards compatibility:

https://github.com/pygfx/pygfx/blob/774a983c0133f458e502b1eef34398f1e8174b93/pygfx/objects/_more.py#L24-L27

Is this safe to remove?

Reply:
I'm leaning towards forcing `Background(None, material)`, because all world objects specify geometry and material in that order. Let's not spend to much time on this though, since the background is a bit up in the air anyway.

Reply:
okay. in this case I will rephrase the docstring instead of removing it.